incubator/jaeger/README.md

```diff
@@ -277,6 +278,8 @@ The following table lists the configurable parameters of the Jaeger chart and th
 | `storage.cassandra.host`                 | Provisioned cassandra host          |  `cassandra`                             |
 | `storage.cassandra.password`             | Provisioned cassandra password  (ignored if storage.cassandra.existingSecret set)     |  `password`                              |
 | `storage.cassandra.port`                 | Provisioned cassandra port          |  `9042`                                  |
+| `storage.cassandra.tls.enabled`          | Provisioned cassandra TLS connection enabled |  `false`                                    |
+| `storage.cassandra.tls.secretName`       | Provisioned cassandra TLS connection existing secret name (possible keys in secret: `ca_cert.pem`, `client_key.pem`, `client_cert.pem`, `cqlshrc`, `commonName`) |  ``                                      |
```

Comment:
nit:  required keys: `key` , `cert` and `ca`

I don't think we need to add `commonName` nor `cqlshrc` in the secret

Reply:
The keys on the secret should be the file names, shouldn't they?

```
apiVersion: v1
kind: Secret
metadata:
  name: cassandra-tls-secret
stringData:
  "client-cert.pem": |
    [...]
```

Reply:
good point

Reply:
Could you please resolve the conflict in Char.yaml?

Reply:
Done

Reply:
Thanks!

Reply:
Would it be possible to include a quick how to create a tls secret with all those keys (including cqlshrc) in the README ? That would be great.

Reply:
Done @naseemkullah